dotnet/docs/EXPERIMENTS.md

```diff
@@ -39,6 +39,7 @@ You can use the following diagnostic IDs to ignore warnings or errors for a part
 | SKEXP0010 | OpenAI parameters | | | | | |
 | SKEXP0010 | OpenAI chat history extension | | | | | |
 | SKEXP0010 | OpenAI file service | | | | | |
+| SKEXP0011 | OpenAI message identity | | | | | |
```

Comment:
We are not doing fine grained experimental flags anymore as it's to inconvenient for developers. This can be included under `SKEXP0001`.